development/generate-lavamoat-policies.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+# Generate LavaMoat policies for the extension background script for each build
+# type.
+# ATTN: This may tax your device when running it locally.
+concurrently --kill-others-on-fail -n main,beta,flask \
+  "WRITE_AUTO_POLICY=1 yarn dist" \
```

Comment:
Is this safe to run concurrently? All three of these builds will write to the `dist` directory, so I'd expect these to be overwriting each other.

Reply:
Hm. For the purposes of validating the LavaMoat config, it doesn't matter what's in the `dist` directory or whether files are being overwritten. The build processes just need to complete normally. We could really use a `--no-emit` option, actually.

My question is whether this could interfere with any of the other build scripts we run, if they also write or read from `dist`?

Reply:
My concern was that LavaMoat itself might be generating the policy files from-disk. So the version of the build it was generating a policy for might be reliant on this race condition of which build finishes when.

But on second thought... that's not how any of this works. LavaMoat runs on the JavaScript while it's still in stream form, before it gets written to disk. At least I'm pretty sure. So yeah you're probably right.

And I agree, a `--no-emit` option would be useful here.

Reply:
im not sure if:
- `clean` is safe (removes and creates the dir) 
- inpage/contentscript is safe (creates the inpage bundle and puts it in the contentscript bundle)
but if it works it works